.github/workflows/build-appimage.yml

```diff
@@ -0,0 +1,81 @@
+name: Build FLINT AppImage
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
Add a `workflow_run` as well here to run the workflow manually. Is it possible to make a dev release in case of `workflow_run` while the main release should only be done in case a tag has been pushed?

Reply:
I'm assuming you mean `workflow_dispatch` used to run actions manually and not `workflow_run` which is used to run a workflow only when a certain workflow finishes before it.

In that case - we can provide a custom input variable and conditionally do a dev release in the `release` job. It'd look something like this:

```yaml
on:
  workflow_dispatch:
    inputs:
      dev:
        description: 'Is this a dev release?'
        type: boolean
        required: true

...
...
...


      - name: Create dev release
        if: "${{ github.event.inputs.dev }}"
        uses: ncipollo/release-action@v1
        with: 
          allowUpdates: true
          name: Dev release
          prerelease: False
          artifacts: "/home/runner/FLINT-*.AppImage"
          token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Oh, yes! My bad. It looks good to me, thanks!